lib/application/pages/home/collections/update/update_collection_page.dart

```diff
@@ -79,8 +78,14 @@ class _UpdateCollectionContents extends ConsumerWidget {
             ],
             child: UpdateCollectionDetails(),
           );
+
         case _Segment.memos:
-          return UpdateCollectionMemos();
+          return ProviderScope(
```

Comment:
You said using keys worked, but I don't see them here, am I missing something?

Reply:
My bad, forgot to commit the changes